.vscode/settings.json

```diff
@@ -3,5 +3,5 @@
   "editor.formatOnSave": true,
   "prettier.requireConfig": true,
   "coverage-gutters.showLineCoverage": true,
-  "cSpell.words": ["skyux", "blackbaud"]
+  "cSpell.words": ["skyux", "blackbaud", "tabset", "tablist"]
```

Comment:
I'm not familiar with this configuration. What extension is this?

Reply:
It is the `Code Spell Checker` extension that Steve recommended the other day.